static/pages/home/home.html

```diff
@@ -45,37 +39,27 @@ <h4 class="md-headline oppia-foundation-headline">
                       home-page-description-text-box">
             <md-card-title class="oppia-foundation-no-horizontal-padding">
               <h1>
-                We&rsquo;re building a global classroom
+                Students from every corner of the world deserve access to 
+                engaging, personalized education.              
               </h1>
             </md-card-title>
-            <p></p>
-            <h3 class="md-headline oppia-foundation-headline">
-              Students from every corner of the world deserve access to 
-              engaging, personalized education.
-            </h3>
           </div>
         </md-card-content>
       </div>
     </div>
   </md-card>
-  <md-card layout-fill
-           class="oppia-foundation-vertical-padding
-                  oppia-foundation-horizontal-padding">
-    <md-card-content layout="row"
+  <md-card layout-fill class="oppia-foundation-vertical-padding
+                              oppia-foundation-horizontal-padding">
+    <md-card-content layout="row" 
                      layout-align="center"
                      class="oppia-foundation-no-padding
                             home-page-description-text-box
                             home-page-display-size">
-      <md-content layout="column"
-                  class="oppia-foundation-white-background
-                         oppia-foundation-center-align-text">
-        <p>
-          More than 250 million children worldwide have not achieved basic 
-          literacy and numeracy.
-        </p>
-        <p class="oppia-foundation-green-text">
-          The Oppia Foundation is working to change that.
-        </p>
+      <md-content layout="column" 
+                  class="oppia-foundation-white-background 
+                        oppia-foundation-center-align-text">
+        <p>More than 1,000,000 children worldwide have not yet achieved basic literacy and numeracy.</p>
```

Comment:
@seanlip Is this correct? 250m down to just 1m?

Reply:
Ah okay, my bad.